boards/arm/cc1352r_launchpad/support/openocd.cfg

```diff
@@ -0,0 +1 @@
+source [find board/ti_cc26x2_launchpad.cfg]
```

Comment:
It feels more appropriate to be using ti_cc13x2_launchpad.cfg instead for the cc1352r

Reply:
When I try to run 'make flash', the code loads and runs fine, but I am seeing this sort of error messages:

```

    TargetName         Type       Endian TapName            State       
--  ------------------ ---------- ------ ------------------ ------------
 0* cc13x2.cpu         cortex_m   little cc13x2.cpu         running
Info : JTAG tap: cc13x2.jrc tap/device found: 0x3bb4102f (mfg: 0x017 (Texas Instruments), part: 0xbb41, ver: 0x3)
Info : JTAG tap: cc13x2.cpu enabled
Error: Debug regions are unpowered, an unexpected reset might have happened
Error: JTAG-DP STICKY ERROR
Error: Could not find MEM-AP to control the core

```

Are you seeing the same? Just wondering if this is expected.

Reply:
Yes, I am seeing the same on both the CC2652 and the CC1352 boards. Not sure what the error is but loading, running and debugging appears fine.